python_modules/libraries/dagster-dg/dagster_dg/config.py

```diff
@@ -54,9 +63,12 @@ def discover_config_file(
 ) -> Optional[Path]:
     current_path = path.absolute()
     while True:
-        config_path = current_path / "pyproject.toml"
-        if config_path.exists() and has_dg_file_config(config_path, predicate):
-            return config_path
+        dg_toml_path = current_path / "dg.toml"
+        pyproject_toml_path = current_path / "pyproject.toml"
+        if dg_toml_path.exists() and has_dg_file_config(dg_toml_path, predicate):
+            return dg_toml_path
+        elif pyproject_toml_path.exists() and has_dg_file_config(pyproject_toml_path, predicate):
+            return pyproject_toml_path
```

Comment:
def need to document somewhere that `dg.toml` takes precedence

Reply:
will add comment but yes also in our docs